src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -100,6 +103,26 @@ export class SignifyApi {
     return this.signifyClient.identifiers().get(name);
   }
 
+  async createRegistry(name: string): Promise<CreateRegistryResult> {
```

Comment:
Thanks for the PR! What is the use case for the wallet creating registries? We hadn't intended for the wallet to need to issue ACDCs, only receive and disclose/present them.

Reply:
My understanding was that the wallet might have a use case for issuing ACDCs to enhance peer-to-peer interactions, allowing wallets to not only receive and present ACDCs but also create and issue them. However, if this goes beyond the intended scope or current use case for the wallet, I appreciate the clarification and am ready to make the necessary adjustments to align with the project's goals and requirements.

Reply:
@KritzHeng This makes sense, and it's a possibility. But I would leave this until a later time for when we have identified those use cases (and when the wallet is more complete for receiving/disclosing).

In general ACDCs will primarily used to give reputation to an identifier _publicly_ so that they can disclose it to a 3rd party based on a trust anchor or chain of trust between our wallet and the 3rd party, but if you have ideas here please open a [discussion](https://github.com/cardano-foundation/cf-identity-wallet/discussions) item and we can flesh it out more. :)